CHANGELOG.md

```diff
@@ -4,7 +4,37 @@ This file documents all notable changes made to this project since runc 1.0.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [unreleased]
+
+## [1.2.0-rc.1] - 2023-08-10
+
+> How, dear sir, did you cross the flood? By not stopping, friend, and by not
+> straining I crossed the flood
+
+This new runc release includes all the changes from the v1.1 patch releases.
+
+### Compatibility
+
+* This release requires Go 1.20.x or Go 1.19.x (#3718)
+
+### Added
+
+ * Support idmap mounts as specified in the OCI runtime-spec v1.1.0. Currently
+   the mount mappings need to be identical to the mappings used in the user
+   namespace section. (#3717)
+ * The `runc features` command is no longer experimental. (#3861)
```

Comment:
This one should probably be in the “Changed” section

Reply:
Right, fixed. Thanks!